### PR TITLE
added ssl to config options

### PR DESCRIPTION
**Pull request by DG4227**

### CODE REVIEW

File: xapix/values.yaml [hunk 1]
```diff
@@ -119,6 +123,10 @@ shisa:
   secret_key_base:  # openssl rand -hex 64
   api_secret:       # openssl rand -hex 10
 
+  # force_ssl forces phoenix to redirect all incoming http requests to https, prevents cookies from being sent with http requests,
```
Comment (DG4227) [inline xapix/values.yaml]: should be "forces shisa"

File: xapix/charts/phoenix/values.yaml [hunk 2]
```diff
@@ -5,10 +5,10 @@ image:
 
 annotations: {}
 replicaCount: 1
+force_ssl: 'true'
```
Comment (mreinsch) [inline xapix/charts/phoenix/values.yaml]:
```suggestion
force_ssl: false
```

File: xapix/charts/shisa/values.yaml [hunk 3]
```diff
@@ -5,8 +5,8 @@ image:
 
 annotations: {}
 replicaCount: 1
+force_ssl: 'true'
```
Comment (mreinsch) [inline xapix/charts/shisa/values.yaml]:
```suggestion
force_ssl: false
```

**Comment by DG4227** [comment]
> with this, please review the check in the deployment file to make sure FORCE_SSL isn't set if this is null / false

how could I do this? some kind of local deploy?

File: xapix/README.md [hunk 4]
```diff
@@ -118,6 +118,9 @@ helm install \
 
 When using AWS SNS to send emails, the kubernetes nodes need to be authorized to perform `ses:SendRawEmail`. You can attach this permission to the node instances when using [kops](https://github.com/kubernetes/kops/blob/master/docs/iam_roles.md#adding-additional-policies) for instance.
 
+##### SSL Considerations
+
+If you're using ELB (Elastic Load Balancer) for SSL termination with Nginx as your ingress, you must ensure the `use-forwarded-headers` and `ssl-redirect` options are properly set in your ingress configuration. Documentation for those can be found ![here](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#use-forwarded-headers) and ![here](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#ssl-redirect).
```
Comment (mreinsch) [inline xapix/README.md]:
```suggestion
If you're using ELB (Elastic Load Balancer) for SSL termination with Nginx as your ingress, you must ensure that `use-forwarded-headers` is enabled and `ssl-redirect` is disabled in your ingress configuration. Documentation for those can be found ![here](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#use-forwarded-headers) and ![here](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#ssl-redirect).
```

**Comment by mreinsch** [comment]
> how could I do this? some kind of local deploy?

we already did that, we're only setting FORCE_SSL to 1 if this is true.

**Comment by DG4227** [comment]
> > how could I do this? some kind of local deploy?
> 
> we already did that, we're only setting FORCE_SSL to 1 if this is true.

Ah - I thought you meant literally confirm that the conditional works properly if the value is false/null. Ok great.

**Pull request merged**